Modules/storage.rb

```diff
@@ -0,0 +1,21 @@
+class Storage
+  @file_extension = 'json'
+  @base_url = './db/'
+
+  def self.save_data(class_name, object)
+    # Create the directory
+    Dir.mkdir(@base_url)
+
+    file_path = "#{@base_url}#{class_name}.#{@file_extension}"
+    File.write(file_path, JSON.generate(object))
+  end
+
+  def self.load_data(class_name)
+    file_path = "#{@base_url}#{class_name}.#{@file_extension}"
+
+    return [] unless File.exist?(file_path)
+
+    data_from_file = File.read(file_path)
+    JSON.parse(data_from_file, create_additions: true)
+  end
```

Comment:
Your storage is functionality is still not working properly. After correcting some of the typo errors I still get some errors as shown in the images below :point_down: Kindly check and make sure everything is working and data is being saved to the json file. :wink: 

![Screenshot from 2023-01-18 17-10-12](https://user-images.githubusercontent.com/65233860/213250096-e1ba01dd-69e0-458b-b5f0-e049e680de66.png)

After correcting the typos and creating books, :point_down: 

![Screenshot from 2023-01-18 17-10-27](https://user-images.githubusercontent.com/65233860/213250126-9514824b-dbf6-4eb9-9221-868a78c0c050.png)


![Screenshot from 2023-01-18 17-10-43](https://user-images.githubusercontent.com/65233860/213250139-193d191d-2481-4647-afe8-d77f0193e85d.png)